lib/tasks/migrate.js

```diff
@@ -24,12 +24,12 @@ module.exports = function runMigrations(context) {
     // If we're using sqlite and the ghost user owns the content folder, then
     // we should run sudo, otherwise run normally
     if (shouldUseGhostUser(contentDir)) {
-        const knexMigratorPath = path.resolve(__dirname, '../../node_modules/.bin/knex-migrator-migrate');
+        const knexMigratorPath = path.resolve(context.instance.dir, 'current/node_modules/.bin/knex-migrator-migrate');
         knexMigratorPromise = context.ui.sudo(`${knexMigratorPath} ${args.join(' ')}`, {sudoArgs: '-E -u ghost'});
     } else {
         knexMigratorPromise = execa('knex-migrator-migrate', args, {
             preferLocal: true,
-            localDir: __dirname
+            localDir: path.join(context.instance.dir, 'current')
```

Comment:
Short understanding question. `execa` extends the PATH and so it's able to execute the local installation inside `current/node_modules`, correct?

Reply:
yup